src/zenml/orchestrators/step_runner.py

```diff
@@ -637,6 +642,16 @@ def _store_output_artifacts(
 
             # Get full set of tags
             tags = step_context.get_output_tags(output_name)
+            if tags is not None:
```

Comment:
`tags` will never be `None` as that method returns a `List[str]`, so this whole if condition can be removed.

Reply:
Removed the condition.